src/sagas/backend.ts

```diff
@@ -266,6 +346,104 @@ async function postAnswer(
   return resp
 }
 
+/**
+ * GET /grading
+ * @returns {Array} GradingOverview[]
+ */
+async function getGradingOverviews(tokens: Tokens): Promise<GradingOverview[] | null> {
+  const response = await request('grading', 'GET', {
+    accessToken: tokens.accessToken,
+    refreshToken: tokens.refreshToken,
+    shouldRefresh: true
+  })
+  if (response && response.ok) {
```

Comment:
`if (response)` is sufficient. Non-null response implies `response.ok`. [Reference](https://github.com/source-academy/cadet-frontend/blob/c61fd6009f480bccc4ed06470a364e5ae9c98b4b/src/sagas/backend.ts#L482).